api/app/kubernetes.py

```diff
@@ -22,7 +21,7 @@ def get(cls) -> str:
 
 
 def get_incluster_namespaces() -> str:
```

Comment:
We are now returning a list, so the type annotation should reflect this (as well as the static method above).